modifiers/caliper-cuda/modifier.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from ramble.modkit import *
+from ramble.mod.benchpark.caliper import Caliper as CaliperBase
+
+class CaliperCuda(CaliperBase):
```

Comment:
Please remove from this PR, lets only have the platform independent modifier for now.